prime-select

```diff
@@ -150,8 +159,17 @@ class Switcher(object):
         if not os.path.isfile(self._nvidia_kms_file):
             self._enable_kms()
 
-    def _blacklist_nvidia(self):
-        blacklist_text = '''# Do not modify
+    def _blacklist_nvidia(self, keep_nvidia_modules=False):
+        blacklist_text = ""
+        if keep_nvidia_modules:
```

Comment:
In this case, we should not blacklist any of the nvidia modules. I will make sure that gpu-manager removes the additional output class that causes Nvidia to be used with X in this case.

Reply:
glad to know that there will be a new complete solution landing in the gpu manager.

fix accordingly. please check the newly pushed branch.